rita/src/rita_common/dashboard/network_endpoints.rs

```diff
@@ -14,10 +14,11 @@ use super::{Dashboard, GetOwnInfo, GetWifiConfig, NodeInfo, OwnInfo, SetWifiConf
 
 use rita_common::dashboard::GetNodeInfo;
 use rita_common::dashboard::WifiInterface;
-use settings::StatsServerSettings;
 
+
+#[allow(unused)] //Called in library in client.rs, not identified as such
 pub fn get_wifi_config(
```

Comment:
ok looked into this more, look at client and exit.rs where these are pulled in (under dashboard) notice that it's not pulled in in exit. The only reason it's not recognized as called is because the library provides some syntactic sugar that isn't an explicit function call but a reference to that function. I think if there's ever a justification for using ignore unused this is it. Because it's just incorrect info.

Reply:
The above is evident in `rita/src/exit.rs`

Reply:
Think we should move this into rita_client/dashboard(new folder)/network_endpoint.rs
(rust unused warnings are pretty good in general, never seen a false positive/negative except for multi-binary stuff, but even then it is valid, just not what you would expect)